mediaflow_proxy/utils/cache_utils.py

```
@@ -1,376 +1,404 @@
+                await f.write(metadata_bytes)
+                await f.write(data)
+
+            await aiofiles.os.rename(temp_path, file_path)
+            return True
+
+        except Exception as e:
+            logger.error(f"Error writing to cache: {e}")
+            try:
+                await aiofiles.os.remove(temp_path)
+            except:
+                pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)